tests/utils/local-network-utils/local_network_setup.go

```diff
@@ -80,12 +82,15 @@ func SetupNetwork(warpGenesisFile string) {
 	// Name 10 new validators (which should have BLS key registered)
 	var subnetANodeNames []string
 	var subnetBNodeNames []string
-	for i := 1; i <= 10; i++ {
+	var subnetCNodeNames []string
```

Comment:
I think `validate_churn` should be updated to issue txs to the propose vm for all subnets. Also the node ids collide from 15-19 right now

Reply:
I think I tried it from 16 to 20, but it still get error. I will double check.